lib/node_modules/@stdlib/blas/ext/base/dapxsumors/include/stdlib/blas/ext/base/dapxsumors.h

```diff
@@ -31,7 +31,12 @@ extern "C" {
 /**
 * Adds a scalar constant to each double-precision floating-point strided array element and computes the sum using ordinary recursive summation.
 */
-double stdlib_strided_dapxsumors( const int64_t N, const double alpha, const double *X, const int64_t stride );
+double API_SUFFIX(c_dapxsumors)( const CBLAS_INT N, const double alpha, const double *X, const CBLAS_INT strideX );
```

Comment:
Same comment as in the other PR. Namely, IMO we should stick with the `stdlib_strided_` naming convention.